skin.json

```diff
@@ -209,6 +209,9 @@
     },
     "tooltips":{
       "enabled":false
+    },
+    "qualitySelection":{
```

Comment:
Can we add this to the schema?

Reply:
Where is the schema located?

Reply:
It's the `skin-schema.json` file. It's used for docs and validation.

Reply:
Done!